stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -148,12 +148,8 @@ function _dump_function(@nospecialize(f), @nospecialize(t), native::Bool, wrappe
     end
     # get the MethodInstance for the method match
     world = typemax(UInt)
-    meth = which(f, t)
```

Comment:
This isn't quite the same query, as this was subtyping, while the other is type-intersection:
```
julia> which(+, (Integer, Integer))
+(a::Integer, b::Integer) in Base at int.jl:921

julia> methods(+, (Integer, Integer))
# 8 methods for generic function "+":
[1] +(x::T, y::T) where T<:Union{Int128, Int16, Int32, Int64, Int8, UInt128, UInt16, UInt32, UInt64, UInt8} in Base at int.jl:87
[2] +(c::Union{UInt16, UInt32, UInt64, UInt8}, x::BigInt) in Base.GMP at gmp.jl:528
[3] +(c::Union{Int16, Int32, Int64, Int8}, x::BigInt) in Base.GMP at gmp.jl:534
[4] +(x::BigInt, y::BigInt) in Base.GMP at gmp.jl:479
[5] +(x::BigInt, c::Union{UInt16, UInt32, UInt64, UInt8}) in Base.GMP at gmp.jl:527
[6] +(x::BigInt, c::Union{Int16, Int32, Int64, Int8}) in Base.GMP at gmp.jl:533
[7] +(x::Bool, y::Bool) in Base at bool.jl:92
[8] +(a::Integer, b::Integer) in Base at int.jl:921
```